packages/salesforcedx-utils-vscode/src/cli/diffResultParser.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { extractJsonObject } from '../helpers';
+
+export interface DiffSuccessResponse {
```

Comment:
I think it'd be good to add some documentation here, like what's currently in MetadataInfo.

Reply:
That's a good idea, let me add something.